chart/compass/templates/tests/external-services-mock/auditlog-secret.yaml

```diff
@@ -7,7 +7,9 @@ metadata:
 type: Opaque
 data:
   {{ .Values.global.externalServicesMock.auditlog.secret.clientIdKey }}: {{ "client_id" | b64enc | quote }}
-  {{ .Values.global.externalServicesMock.auditlog.secret.clientSecretKey }}: {{ "client_secret" | b64enc | quote }}
-  {{ .Values.global.externalServicesMock.auditlog.secret.tokenUrlKey }}: {{ printf "http://compass-external-services-mock.%s.svc.cluster.local:8080/secured" .Release.Namespace | b64enc | quote }}
+  {{ .Values.global.externalServicesMock.auditlog.secret.clientSecretKey }}: {{ "client_secret" | b64enc | quote }} # TODO:: remove
```

Comment:
leftover?

Reply:
Actually, it's not a leftover. We will keep the standard oauth with client id and secret